code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/message/views/Message.kt

```diff
@@ -101,7 +104,8 @@ internal fun Message(
          */
         Dialog(
             properties = DialogProperties(
-                usePlatformDefaultWidth = false,
+                usePlatformDefaultWidth = Build.VERSION.SDK_INT >= 35,
```

Comment:
[Edge-to-edge enforcement](https://developer.android.com/about/versions/16/behavior-changes-16) is applied only when the _app_ targets Android 35 or above. If the app targets 34 and is running on Android 16, edge-to-edge is not seen unless the app activity specifically adds `enableEdgeToEdge()` call.
Since `Build.VERSION.SDK_INT` provides the SDK version of the software currently running on this hardware device, you will likely have to additionally use something like `val packageInfo: PackageInfo = context.packageManager.getPackageInfo(context.packageName, 0)`.

Reply:
1. With this particular [solution](https://issuetracker.google.com/issues/246909281#comment19), `usePlatformDefaultWidth` needs to be set to true, I'm not sure why. Without the setting I see some padding on the top on the screen

![Screenshot_20250609_125405](https://github.com/user-attachments/assets/c0f5a410-0aa7-4498-9f4c-6e81efabc3a1)

![Screenshot_20250609_125058](https://github.com/user-attachments/assets/2e447f3d-3840-4d3a-9a01-9d7de15bf52a)

2. Will make the change